### DISCUSSION TITLE
No bookmarks shown, only white stripes.

### DISCUSSION THREAD

**Discussion by akisgd** [nextcloud/bookmarks#1933] [General]
Hello, suddenly the bookmarks app doesn't show any bookmarks only white empty stripes (when I'm using dark theme).
At the left side the number of the bookmarks looks OK. The tags and the bookmarks at each tag looks OK. But when I'm selecting any tag or all bookmarks nothing shown to the right part. Only many white empty stripes.
Has anyone had something similar. I don't know what to do. I removed and reinstalled the app but nothing changed. Maybe the db has been corrupted. The procedure of remove and reinstall the app doesn't delete the db.

Same behavior on PC and android browser. 

![Screenshot_20221124_200421](https://user-images.githubusercontent.com/7311967/203845579-4bd8da06-5787-47aa-ac07-7a9914dce54f.jpg)

![Screenshot_20221124_200356](https://user-images.githubusercontent.com/7311967/203845611-6ed8d295-28e9-4559-bdfc-ca379d9ceb73.jpg)

**Comment by marcelklehr** [comment]
If you just upgraded to v11 your mysql version may be too old

**Reply by akisgd** [reply]
Thank you for your reply. I haven't made any changes prior this problem arise. Trying to resolve it i have just upgraded nextcloud installation. It didn't made any difference... The same problem.
My mysql is... 
mysql  Ver 15.1 Distrib 10.1.48-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

**Reply by marcelklehr** [reply]
In that case your issue is described in #1914 which doesn't have a solution, yet, sadly.

**Reply by marcelklehr** [reply]
@akisgd would you mind executing my suggestion at https://github.com/nextcloud/bookmarks/issues/1914#issuecomment-1336445189 ?

**Reply by akisgd** [reply]
Hello, i'm sorry but i can't perform this action. Trying to resolve my problem i reset everyrhing and i started with a new nextcloud installation on new vps server and new bookmarks app installation. Initially i got very disappointed because i had exact the same issue. I restored my backed up database but there was no entry actually present at the application. Then i saw a message at the nextcloud update page regarding maria db update and i updated to my new installation at maria db 10.5 and the problem gone. Now everything is working as it should.
